lib/kb_stringtie/Utils/StringTieUtil.py

```diff
@@ -201,23 +218,23 @@ def _create_gtf_annotation_from_genome(self, genome_ref, result_directory):
             raise ValueError(
                 "Genome at {0} does not have reference to the assembly object".format(
                     genome_ref))
-        print contig_id
+        print(contig_id)
         log("Generating GFF file from Genome")
         try:
             ret = self.au.get_assembly_as_fasta({'ref': contig_id})
             fa_output_file = ret['path']
 
             shutil.copy(fa_output_file, result_directory)
-            fa_output_name = os.path.basename(fa_output_file)
-            fa_output_file = os.path.join(result_directory, fa_output_name)
+            #fa_output_name = os.path.basename(fa_output_file)
+            #fa_output_file = os.path.join(result_directory, fa_output_name)
 
-            mapping_filename = c_mapping.create_sanitized_contig_ids(fa_output_file)
+            #mapping_filename = c_mapping.create_sanitized_contig_ids(fa_output_file)
 
             # get the GFF
             ret = self.gfu.genome_to_gff({'genome_ref': genome_ref,
                                           'target_dir': result_directory})
             genome_gff_file = ret['file_path']
-            c_mapping.replace_gff_contig_ids(genome_gff_file, mapping_filename, to_modified=True)
+            #c_mapping.replace_gff_contig_ids(genome_gff_file, mapping_filename, to_modified=True)
```

Comment:
I removed this contig renaming step because I found no cases where it's needed and it was causing confilcts.

Reply:
Let's go ahead and delete the commented lines.